package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@rdkit/rdkit",
+  "version": "2021.3.1-beta.14",
+  "description": "JavaScript distribution of RDKit - a collection of cheminformatics and machine-learning software written in C++ and Python.",
+  "main": "Code/MinimalLib/dist/RDKit_minimal.js",
+  "scripts": {
+    "build": "bash Code/MinimalLib/scripts/build_rdkitjs.sh",
+    "prepublish": "bash Code/MinimalLib/scripts/npm_prepublish.sh",
+    "postpublish": "bash Code/MinimalLib/scripts/npm_postpublish.sh",
+    "test": "echo \"Tests are run during the docker build. See https://github.com/rdkit/rdkit/blob/master/Code/MinimalLib/docker/Dockerfile .\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rdkit/rdkit.git"
+  },
+  "keywords": [
+    "javascript",
+    "c-plus-plus",
+    "cheminformatics",
+    "rdkit",
+    "wasm"
+  ],
+  "author": "Rational Discovery LLC, Greg Landrum, Julie Penzotti and others",
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/rdkit/rdkit/issues"
+  },
+  "homepage": "https://github.com/rdkit/rdkit/blob/master/Code/MinimalLib/README.md",
+  "files": [
+    "license.txt",
+    "package.json",
+    "ReleaseNotes.md",
+    "README.md",
+    "Code/MinimalLib/dist/demo.html",
```

Comment:
Good idea, I removed the old demo and replaced it with the GettingStarted page. We can see it as the "official" demo now.

Reply:
I'm not a fan of changing the repo to overwrite the old `demo.html` - which provided a minimal, yet not useless, demo of how the package can be used - with a copy of the documentation html file.
I would prefer including the documentation file in the package instead of the demo.html file (or in addition to it), but not getting rid of the demo file itself.

Reply:
Ok, will make the change